src/index.css

```diff
@@ -1,31 +1,137 @@
 body {
-  margin: 10px;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  margin: 0;
+  background: #f8f8f8;
+}
+
+.navbar {
+  height: 5.938rem;
+  background: white;
+  padding-top: 30px;
+  padding-left: 30px;
+}
+
+.navlink {
+  text-decoration: none;
+  margin-left: 20px;
+}
+
+#logo {
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.nav {
+  width: 3.688rem;
+  height: 1rem;
+  font-family: Montserrat, sans-serif;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.navbar__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: white;
+}
+
+#profile-pic {
+  width: 50px;
+  height: 50px;
+  border-radius: 100%;
+  margin-right: 30px;
 }
 
 .book__container {
-  margin-bottom: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: white;
+  margin: 50px;
+  padding: 20px;
+}
+
+.percentage__container {
+  display: flex;
+  align-items: center;
+}
+
+#progress-circle {
+  width: 100px;
+  height: 100px;
+  margin-right: 10px;
+}
+
+#update__button {
+  width: 150px;
+  background: #0290ff;
+  color: white;
+  height: 30px;
+  cursor: pointer;
+}
+
+.form__container {
+  background: white;
+  margin: 50px;
+  padding: 40px;
+}
+
+.form__container input {
+  width: 150px;
+  height: 20px;
+  margin-right: 10px;
+}
```

Comment:
- [ ] Please make you form match the original design 

Original design 👇 

![image](https://user-images.githubusercontent.com/75126481/179218327-f2c21069-d4fd-4363-bb5d-647876ee0b08.png)

Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/179218483-2be1569a-9531-4de5-b1cd-b05f7fc60bbe.png)